crates/fs/src/fs.rs

```diff
@@ -245,9 +245,8 @@ impl Fs for RealFs {
         #[cfg(unix)]
         let inode = metadata.ino();
 
-        // todo!("windows")
         #[cfg(windows)]
-        let inode = 0;
+        let inode = file_id(path);
```

Comment:
We already open the file above using smol, can we restructure this code somehow to not open the file twice?